src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/Hop.tsx

```diff
@@ -25,16 +33,56 @@ export const Hop = ({
   slippageDecimalPercentage,
   networkFeeFiatPrecision,
   protocolFeeFiatPrecision,
+  title,
+  hopIndex,
+  txStatus,
+  isOpen,
+  onToggleIsOpen,
 }: {
   steps: StepperStep[]
   activeStep: number
   slippageDecimalPercentage: string
   networkFeeFiatPrecision: string
   protocolFeeFiatPrecision: string
+  title: string
+  hopIndex: number
+  txStatus?: TxStatus
+  isOpen: boolean
+  onToggleIsOpen: () => void
 }) => {
   const backgroundColor = useColorModeValue('gray.100', 'gray.750')
   const borderColor = useColorModeValue('gray.50', 'gray.650')
 
+  const timeEstimate = '4m'
+  const timeRemaining = '3:35'
+
+  const rightComponent = useMemo(() => {
+    switch (txStatus) {
+      case undefined:
+      case TxStatus.Unknown:
+        return <RawText>{timeEstimate}</RawText>
+      case TxStatus.Pending:
+        return <RawText>{timeRemaining}</RawText>
+      case TxStatus.Confirmed:
+        return (
+          <Box width='auto'>
+            <IconButton
+              aria-label='expand'
+              variant='link'
+              p={4}
+              borderTopRadius='none'
+              colorScheme='blue'
+              onClick={onToggleIsOpen}
+              width='full'
+              icon={isOpen ? <ChevronUpIcon boxSize='16px' /> : <ChevronDownIcon boxSize='16px' />}
```

Comment:
Memoize left and right side - memo eslint rule isn't smart enough to catch these :(